scripts/deploy_example_gcp

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPTS_DIR/common"
+
+"$SCRIPTS_DIR/build_example" "hello_world"
+
+bazel_build_flags+=(
+    '--config=clang'
+)
+
+# Prerequisites:
+# - Google Cloud SDK
+# - Environment variable GCP_PROJECT_ID: id of the GCP project to use (eg oak-vm-testing).
+# - Environment variable GCP_ACCOUNT_FILE: path to the exported key for the service
+#   account that will be used to connect to GCP.
+# - Service account must be in the following IAM roles:
+#   - Compute Instance Admin (v1)
+#   - Compute Security Admin
+#   - Service Account User 
+
+# Create a minimal ubuntu linux image on the GCP project.
+# The image will be named oak-gcp-hello-world.
+bazel run "${bazel_build_flags[@]}" //examples/hello_world/gcp:deploy-oak-gcp-hello-world
+
+# Authenticate as the service account
+gcloud auth activate-service-account --project=$GCP_PROJECT_ID --key-file=$GCP_ACCOUNT_FILE
```

Comment:
(nit) quote and use `${` notation for variables.

https://google.github.io/styleguide/shell.xml?showone=Variable_expansion#Variable_expansion